src/PaymentMethodCollectElement.res

```diff
@@ -327,20 +345,22 @@ let make = (~integrateError, ~logger) => {
           </div>
         </div>
       </div>
-      <div className="mt-[40px]">
-        {switch secondsUntilRedirect {
-        | Some(seconds) =>
-          React.string("Redirecting in " ++ seconds->Int.toString ++ " seconds ...")
-        | None => React.null
-        }}
-      </div>
+      {switch secondsUntilRedirect {
+      | Some(seconds) =>
+        <div className="mt-10">
+          {React.string("Redirecting in " ++ seconds->Int.toString ++ " seconds ...")}
```

Comment:
does secondsUntilRedirect update every second?
Might be a better UX if you do

Reply:
Yes, it updates every second